File: js/stockPicker.js
exports.picker = function(prices) {
    let largestProfit = 0

    for(i=0; i<prices.length; i++) {
        for(j=1; j<prices.length; j++) {
            if(i<j) {
                let profit = prices[j] - prices[i]
                    if(profit > largestProfit) {
                        largestProfit = profit
                        buySell = [i, j]
                    }
            }
        }
    }

    return buySell
}

// console.log(exports.picker([17,3,6,9,15,8,6,1,10]))

File: python/stock_picker.py
def picker(prices):
    largest_profit = 0

    for i in range(len(prices)):
        for j in range(1, len(prices)):
            if i<j:
                profit = prices[j]-prices[i]
                if profit > largest_profit:
                    largest_profit = profit
                    buy_sell = [i, j]

    return buy_sell